.github/send_alerts_to_slack.py

```diff
@@ -0,0 +1,95 @@
+"""
+
```

Comment:
Please rename this script to be more meaningful/specific — maybe something like goodtables_summarize ? goodtables_report ?